private_gpt/ui/ui.py

```diff
@@ -206,6 +206,10 @@ def _set_system_prompt(self, system_prompt_input: str) -> None:
         logger.info(f"Setting system prompt to: {system_prompt_input}")
         self._system_prompt = system_prompt_input
 
+    def _set_model_temperature(self, model_temperature_input: str) -> None:
+        logger.info(f"Setting Model Temperature to: {model_temperature_input}")
+        self._model_temperature = model_temperature_input
```

Comment:
Where do you use `self.model_temperature`? You are setting the property, but not sending it to the server. Are there any changes missing? @Castolus